docs/book/src/providers/v1alpha2-to-v1alpha3.md

```diff
@@ -40,3 +40,7 @@
 ## The `util/restmapper` package has been removed
 
 - Controller runtime has native support for a [DynamicRESTMapper](https://github.com/kubernetes-sigs/controller-runtime/pull/554/files), which is used by default when creating a new Manager.
+
+## The `util.IsControlPlaneMachine` is now accepts empty value of `MachineControlPlaneLabelName` label
```

Comment:
@ncdc updated docs here, ptal :)

Reply:
I think the thing to point out in this document is that if the label is set the machine is determined to be a control plane. It does more than accept empty values, it accepts *any* value for the label.